test/unit/js/persist.test.js

```diff
@@ -285,6 +300,21 @@ test("persist : save state data by key, get state data by key", function() {
 	deepEqual(clonedState, this.data);
 });
 
+$.each([null, undefined, ''], function(i, v) {
+	test("persist : remove state data with value by key", function() {
+		// Given
+		var state = $.persist("TESTKEY");
+		$.persist("TESTKEY", this.data);
+
+		// When
+		var clonedState = $.persist("TESTKEY", v);
+
+		// Then
+		equal(clonedState, v);
+	});
+})
```

Comment:
both tests are exactly same. Is really necessary have duplicated test?
If so, can you find to remove duplication?